libs/types/src/generic.ts

```diff
@@ -2,6 +2,9 @@ import check8601 from '@antongolub/iso8601';
 import { z } from 'zod';
 import { err, ok, Result, wrapException } from './result';
 
+export interface DefinedDictionary<T> {
+  [key: string]: T;
+}
```

Comment:
I have been meaning to define a type like this, thanks!

Reply:
I would have changed the other to `OptionalDictionary` but 🤷‍♂️